MapleServer2/Types/Inventory.cs

```diff
@@ -83,9 +144,12 @@ public class Inventory {
             return item.TrySplit(amount, out removedItem) ? item.Amount : -1;
         }
 
+
```

Comment:
Why is there an extra new line now?

Reply:
?? i didn't add that. its always been there